terraform-modules/aws/route53/record-vpc-endpoint-sftp-transfer/main.tf

```diff
@@ -0,0 +1,42 @@
+
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+
+resource "null_resource" "output-vpc-endpoint-id" {
+  provisioner "local-exec" {
+    command = "aws transfer describe-server --server-id ${var.transfer_server_id} --query 'Server.EndpointDetails.VpcEndpointId' > ${data.template_file.log_name.rendered}"
```

Comment:
It seems extremely to use a provisioner to get vpc id enpdoint, but it is working well. 
aws transfer module in aws and cloud posse doesn't have an output to vpc endpoint id.
Even terraform state has the value, but is a primitive string, so its forbidden to get.
<img width="1103" alt="Screenshot 2023-03-03 at 13 13 06" src="https://user-images.githubusercontent.com/19688747/222806814-747531b8-ed33-46f5-b2dc-eab16c5033fb.png">

Reply:
cloud posse: https://github.com/cloudposse/terraform-aws-transfer-sftp
aws terraform provider: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transfer_server